devices/wholeBodyDynamics/CMakeLists.txt

```diff
@@ -7,8 +7,7 @@ find_package(YARP REQUIRED)
 find_package(ICUB REQUIRED)
 find_package(iDynTree REQUIRED)
 
-list(APPEND CMAKE_MODULE_PATH "${YARP_MODULE_PATH}")
-include(YarpInstallationHelpers)
+list(APPEND CMAKE_MODULE_PATH)
```

Comment:
This is a nop, as it just adds no string to the `CMAKE_MODULE_PATH` variable list, so it can be removed.
```suggestion
```

Reply:
Yes right. My bad.

Reply:
Done in https://github.com/robotology/whole-body-estimators/pull/87/commits/d1caaef7675cfd72cdfcc30d98e867d574302de6